src/components/MediaSettings/MediaSettings.vue

```diff
@@ -107,7 +107,12 @@
 					:device-id="videoInputId"
 					@refresh="updateDevices"
 					@update:deviceId="handleVideoInputIdChange" />
-				<MediaDevicesSpeakerTest />
+				<MediaDevicesSelector kind="audiooutput"
```

Comment:
MediaDevicesSelector has not icon for this defined

Reply:
Wanted to discuss it first before start actually doing something.
For now, see the mock for the reference